librz/core/cmd_descs/cmd_descs.yaml

```diff
@@ -237,9 +237,8 @@ commands:
     summary: Make directories and move files
     type: RZ_CMD_DESC_TYPE_OLDINPUT
   - name: L
-    cname: cmd_plugins
     summary: List, unload, load rizin plugins
-    type: RZ_CMD_DESC_TYPE_OLDINPUT
```

Comment:
You can keep this if you only want to convert *some* `L` subcommands. If you keep `L` as OLDINPUT, all commands that you have not explicitly converted to newshell in `cmd_plugins.yaml` will still go through the old handlers. If, instead, you remove this line as you did, you have to convert *all* `L` subcommands to newshell.